internal/config/legacy/legacy.go

```diff
@@ -0,0 +1,329 @@
+package legacy
+
+import (
+	"net/url"
+	"strings"
+)
+
+// Pre1127 is a pre-1.12.7 config.
```

Comment:
Shouldn't we add here a "pre 1.15" test?

Reply:
It's not a test. But a backwards compat handler. The latest legacy config is the pre-1.15 one.